modules/TestUtils.js

```diff
@@ -20,6 +20,12 @@ export const isObject = (object) =>
   object && !isArray(object) && typeof object === 'object'
 
 /**
+ * Returns true if the given object is an empty object.
+ */
+export const isEmptyObject = (object) =>
+  isObject(object) && Reflect.ownKeys(object).length === 0
```

Comment:
in no way can you assume that `Reflect.ownKeys` is available - if you want to use that, please use a module instead of assuming that the user has polyfilled this method.